package.json

```diff
@@ -25,6 +25,7 @@
     "i18next-browser-languagedetector": "^7.0.0",
     "javascript-time-ago": "^2.5.9",
     "lodash.debounce": "^4.0.8",
+    "lodash.get": "^4.4.2",
```

Comment:
@IhorNesterenko99 , @nmialik , @kdanilenka32 , @wildkiomi why not to use the strict versions in package json? Sometimes teamates have issues relating to specific version of some package, it's very difficult to distinguish the root of the issue sometimes because of different versions of local packages